source/faq/questions/general-questions-faq.md

```diff
@@ -86,8 +88,8 @@ No. But if you're wanting to test parts of your application that are not easily
 
 No. In fact Cypress' architecture is very different from Selenium in a few critical ways:
 
-- Cypress runs in the context of the browser. With Cypress it's much easier to accurately test the browser, but harder to talk to the outside world. In Selenium it's the exact opposite, it runs outside of the browser where your application is running. Although Cypress has a few commands that give you access to the outside world - like {% url `cy.request()` request %}, {% url `cy.exec()` exec %}, and {% url `cy.task()` task %}.
-- With Selenium - aka WebDriver, you either get 100% simulated events (with Selenium RC) or you got 100% native events with Selenium WebDriver. However, with Cypress, you actually get both. For the most part we use simulated events; We can reproduce them with nearly 100% fidelity, they are much faster, and guaranteed to work the same way every time. However we do use automation API's for things like Cookies where we extend outside of the JavaScript sandbox and interact with the underlying browser API's. This gives us flexibility to determine which type of event to use in specific situations. We've yet to add native input events though.
+- Cypress runs in the context of the browser. With Cypress it's easier to inspect what it running in the browser, but harder to talk to the outside world. In Selenium it's the exact opposite. Selenium runs outside of the browser where your application is running. Although Cypress is adding more commands every day that give you access to the outside world - like {% url `cy.request()` request %}, {% url `cy.exec()` exec %}, and {% url `cy.task()` task %}.
+- With Selenium - aka WebDriver, you either get 100% simulated events (with Selenium RC) or you got 100% native events with Selenium WebDriver. However, with Cypress, you get both. For the most part we use simulated events. However we do use automation API's for things like Cookies where we extend outside of the JavaScript sandbox and interact with the underlying browser API's. This gives us flexibility to determine which type of event to use in specific situations. Native event support is on our {% url "roadmap" roadmap %}.
```

Comment:
`Typo`

`or you got` ➡️ `or you get` (Mixed tenses.)

Or possibly rephrased as:

> With Selenium you get either 100% simulated events (with Selenium RC) or 100% native events (with Selenium WebDriver).

Reply:
`Minor` `Style`

`However, with Cypress, you get both.` ➡️ `With Cypress, you get both.`

Because:

* there's a subsequent sentence that starts with "However" in the same paragraph.
* the sentence still reads well which makes one think that the opening "However" was only serving as a [filler word](https://www.writerightwords.com/10-filler-words/) like "just" and "really" often do.

Reply:
`Minor` `Style`

Is it `API's` or `APIs`? If we unfurl the acronyms there'd be no apostrophe&mdash;Application Programming Interfaces&mdash;'cos it's a plural.

https://english.stackexchange.com/posts/921/revisions

Happy to continue to use "API's" if that's your style: just thought I'd ask so that I can be consistent with your preferred style.

Reply:
Thanks, yes, fixed all instances of API's and the other fixes you mentioned. Thanks 👍